garden_ai/garden.py

```diff
@@ -1,27 +1,35 @@
+import json
 import logging
 import os
 from pathlib import Path
+from typing import List, Union
 
+import requests
 from globus_sdk import (
+    AuthAPIError,
+    AuthClient,
     GroupsClient,
-    RefreshTokenAuthorizer,
     NativeAppAuthClient,
-    AuthClient,
-    AuthAPIError,
+    RefreshTokenAuthorizer,
 )
-from typing import List
 from globus_sdk.tokenstorage import SimpleJSONFileAdapter
-
-from garden_ai.models import Garden
+from globus_sdk.scopes import ScopeBuilder
 from pydantic import ValidationError
 
+from garden_ai.models import Garden, Pipeline
+
 logger = logging.getLogger()
 
 
 class AuthException(Exception):
     pass
 
 
+GardenScopes = ScopeBuilder(
```

Comment:
@ any reviewer(s) who would know, please double check that this (the `GardenScopes` class) is roughly the way to go about this? I wasn't familiar with the idiom before stumbling on analogous code in the globus sdk